site/docs/sql/02-job-management.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Job Management
+description: Commands to manage SQL jobs
+---
+
+## CREATE/DROP JOB
+
+Creates a job from a query that is not tied to the client session. When
+you submit an INSERT query, its lifecycle is tied to the client session:
+if the client disconnects, the query is cancelled, even though it
+doesn't deliver results to the client.
+
+If you want to submit a statement that is independent from the client
+session, use the `CREATE JOB` command. Such a query will return quickly
+and the job will run on the cluster. It can also be configured to be
+fault tolerant.
+
+### CREATE JOB Synopsis
+
+```sql
+CREATE JOB [IF NOT EXISTS] job_name
+[ OPTIONS ( option_name 'option_value' [, ...] ) ]
+AS query_spec
+```
+
+- `job_name`: a unique name identifying the job.
+
+- `query_spec`: the query to run by the job. It must not return rows to
+  the client, that is it must not start with `SELECT`. Currently we
+  support `INSERT INTO` or `SINK INTO` queries.
+
+- `option_name`, `option_value`: the job configuration options. The list
+  of options matches the methods in the `JobConfig` class.
+
+#### Example
+
+```sql
+CREATE JOB myJob
+OPTIONS (
+    processingGuarantee 'exactlyOnce',
+    snapshotIntervalMillis '5000',
+    metricsEnabled 'true'
+) AS
+INSERT INTO my_sink_topic
+SELECT * FROM my_source_topic
+```
+
+### DROP JOB Synopsis
+
+```sql
+DROP JOB [IF EXISTS] job_name [WITH SNAPSHOT snapshot_name]
+```
+
+- `IF EXISTS`: don't throw an error if the job doesn't exist
+
+- `WITH SNAPSHOT`: export a named snapshot before cancelling the job
+  (enterprise feature)
+
+## Other job operations
```

Comment:
Instead of having an "other" section, we may move `ALTER JOB` to a job-related section, and introduce a separate snapshot section. It could make the structure a little bit more clear to users.